packages/hardware-trezor/src/transformers/keyPaths.ts

```diff
@@ -0,0 +1,31 @@
+import { BIP32Path } from '@cardano-sdk/crypto';
+import { Cardano } from '@cardano-sdk/core';
+import { CardanoKeyConst, GroupedAddress, util } from '@cardano-sdk/key-management';
+import { TrezorTxTransformerContext } from '../types';
+
+/**
+ * Constructs the hardened derivation path of the payment key for the
+ * given grouped address of an HD wallet as specified in CIP 1852
+ * https://cips.cardano.org/cips/cip1852/
+ */
+export const paymentKeyPathFromGroupedAddress = (address: GroupedAddress): BIP32Path => [
```

Comment:
We should move shared utils to `key-management` to deduplicate with `hardware-ledger`

Reply:
Let's leave this thread as unresolved, so we don't forget :)

Reply:
Has this been done @DominikGuzei ?

Reply:
As far as I understand, this comment has been addressed: https://github.com/input-output-hk/cardano-js-sdk/blob/794cdef5153d35f3ff8866b3f24bde6172ec604e/packages/key-management/src/util/key.ts

However, we identified more areas for refactoring, which were captured under [LW-8775](https://input-output.atlassian.net/browse/LW-8775)

Reply:
Yes, we did the straight forward stuff already and deferred the more complicated refactorings 👍